src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -152,11 +153,6 @@ class TimelineMarkerCanvas extends PureComponent {
           );
           const h: CssPixels = ROW_HEIGHT - 1;
 
-          if (w < 2) {
-            // Skip sending draw calls for sufficiently small boxes.
-            continue;
-          }
-
```

Comment:
As in the previous PR, this is actually not used because of the Math.max calculation above.